src/components/quiz/question-display.tsx

```
@@ -0,0 +1,85 @@
+  question,
+  selectedOption,
+  isAnswered,
+  currentAnswer,
+  onOptionSelect,
+  showCorrect,
+}: QuestionDisplayProps) {
+  return (
+    <div className="space-y-4">
+      <div className="prose max-w-none">
+        <div
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)